platform/jewel/ui/api-dump-experimental.txt

```diff
@@ -63,13 +64,6 @@ f:org.jetbrains.jewel.ui.component.TypographyKt
 f:org.jetbrains.jewel.ui.component.search.HighlightKt
 - *sf:highlightSpeedSearchMatches(androidx.compose.ui.Modifier,androidx.compose.ui.text.TextLayoutResult,org.jetbrains.jewel.ui.component.NodeSearchMatchState,androidx.compose.runtime.Composer,I,I):androidx.compose.ui.Modifier
 - *sf:highlightTextSearch(java.lang.CharSequence,org.jetbrains.jewel.ui.component.NodeSearchMatchState,androidx.compose.runtime.Composer,I,I):androidx.compose.ui.text.AnnotatedString
-*f:org.jetbrains.jewel.ui.component.search.SpeedSearchableLazyColumnKeyActions
-- org.jetbrains.jewel.foundation.lazy.tree.KeyActions
-- sf:$stable:I
-- <init>(org.jetbrains.jewel.foundation.lazy.tree.KeyActions,org.jetbrains.jewel.ui.component.SpeedSearchState):V
-- getActions():org.jetbrains.jewel.foundation.lazy.SelectableColumnOnKeyEvent
-- getKeybindings():org.jetbrains.jewel.foundation.lazy.SelectableColumnKeybindings
-- handleOnKeyEvent-jhbQyNo(java.lang.Object,java.util.List,org.jetbrains.jewel.foundation.lazy.SelectableLazyListState,org.jetbrains.jewel.foundation.lazy.SelectionMode):kotlin.jvm.functions.Function1
```

Comment:
Those changes were not released yet, so we are good with changing them. Just changing the visibility to internal as we should not need it outside